sklearn/ensemble/_hist_gradient_boosting/grower.py

```diff
@@ -134,45 +134,50 @@ class TreeGrower:
 
     Parameters
     ----------
-    X_binned : ndarray of int, shape (n_samples, n_features)
+    X_binned : ndarray of int of shape (n_samples, n_features)
```

Comment:
```suggestion
    X_binned : ndarray of shape (n_samples, n_features), dtype=int
```

Reply:
Here this is actually `np.uint8` define by `X_BINNED_DTYPE`

Reply:
Changed int to np.uint8 in new commit.